ingestion/src/metadata/ingestion/source/connections.py

```diff
@@ -47,3 +48,13 @@ def get_connection(connection: BaseModel) -> Any:
     a service connection pydantic model
     """
     return get_connection_fn(connection)(connection)
+
+
+def kill_active_connections(engine: Engine):
+    """
+    Method to kill the active connections
+    as well as idle connections in the engine
+    """
+    active_conn = engine.pool.checkedout() + engine.pool.checkedin()
```

Comment:
ok now maybe silly comment, but we're now passing this at the `set_inspector` stage.

Is it possible this will ever fail? Should we wrap this on a try/catch and log if something goes wrong? We don't want this cleanup stage to blow up the full ingestion

Reply:
Will add a try catch block her

Reply:
added